matron/src/weaver.c

```diff
@@ -53,7 +53,7 @@
 
 //------
 //---- global lua state!
-lua_State *lvm;
+static lua_State *lvm;
```

Comment:
i did in fact notice this! thanks.

yes i would fully endorse making all things explicitly `static` that can be.